packages/engine/src/gltf/GLTFComponent.tsx

```diff
@@ -82,6 +83,8 @@ export const GLTFComponent = defineComponent({
 
   schema: S.Object({
     src: S.String(''),
+    library: S.Bool(false),
```

Comment:
this shouldn't be necessary. there should only be one loader and it should take all the descriptions in the gltf and load it into the ecs, regardless if it is part of a node hierarchy or not. we don't want different kinds of loaders and different kinds of loading methodologies.